ibm_catalog.json

```diff
@@ -285,9 +304,91 @@
 							"default_value": "__NULL__",
 							"description": "The CRN of an existing KMS instance to use in this solution. If not set, a new KP instance is provisioned.",
 							"required": false
+						},
+						{
+							"key": "existing_en_instance_crn",
+							"type": "string",
+							"default_value": "__NULL__",
+							"description": "The CRN of an existing event notification instance to use in this solution. If not set, a new event notification instance is provisioned.",
+							"required": false
+						},
+						{
+							"key": "elasticsearch_plan",
+							"type": "string",
+							"default_value": "enterprise",
+							"description": "The pricing plan for the Databases for Elasticsearch instance.",
+							"required": false,
+							"options": [
+								{
+									"displayname": "Enterprise",
+									"value": "enterprise"
+								},
+								{
+									"displayname": "Platinum",
+									"value": "platinum"
+								}
+							]
+						},
+						{
+							"key": "existing_discovery_instance",
+							"type": "string",
+							"default_value": "__NULL__",
+							"description": "The CRN of an existing WatsonX SaaS discovery instance to use in this solution. If not set, a new discovery instance is provisioned depending on which plan is selected.",
+							"required": false
+						},
+						{
+							"key": "existing_assistant_instance",
```

Comment:
I think the input names should be consistent here - based on the en model above, existing_assistant_instance_crn is this takes the crn. Same comment for other input variables.

Reply:
we will address these changes in dev repo (to not increase merge conflicts) with new PR. Issue created https://github.ibm.com/GoldenEye/issues/issues/10462